frontend/src/metabase/search/utils/enabled-search-type/index.ts

```diff
@@ -0,0 +1 @@
+export { isEnabledSearchModelType } from "./enabled-search-type";
```

Comment:
This file contains only 1 function.
If there are no plans to add more functions here let's rename the file to `isEnabledSearchModelType`.

Reply:
I've added a unit test file and another function for filtering the search types to keep all of that logic together, so I'll keep it for now